staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.pb.go

```diff
@@ -6964,6 +6964,395 @@ func (m *ReopenContainerLogResponse) XXX_DiscardUnknown() {
 
 var xxx_messageInfo_ReopenContainerLogResponse proto.InternalMessageInfo
 
+// Common options used for checkpointing and restoring.
```

Comment:
Wondering what's the plan for adding more options in the future? Is this the exhaustive list or plan is to start with some basic options and to add more based on the use case and need? Are these options based off of options that are available for criu here - http://manpages.ubuntu.com/manpages/bionic/man8/criu.8.html?